app/views/admin/staches/new.html.erb

```diff
@@ -1,31 +1,2 @@
-<div class="container">
-  <%= form_for([:admin, @stache]) do |f| %>
-
-    <%= render "shared/error_messages", object: f.object %>
-
-    <%= f.text_field :name %>
-    <%= f.label :name %>
-
-    <%= f.text_field :description %>
-    <%= f.label :description %>
-
-    <%= f.number_field :price %>
-    <%= f.label :price %>
-
-    <%= f.text_field :image_url %>
-    <%= f.label :image_url, "Image Url" %>
-    <br>
-    <br>
-
-    <%= f.collection_check_boxes :categories, Category.all, :id, :title, include_hidden: false do |b| %>
-     <div class="categories-checkboxes">
-       <%= b.check_box %>
-       <%= b.label %>
-     </div>
-      <% end %>
-    <%= f.label :categories %>
-    <br>
-    <br>
-    <%= f.submit "Create New Stache", class: "btn light-green darken-4"%>
-  <% end %>
-</div>
+<% provide(:button_text, "Create New Stache") %>
+<%= render "form" %>
```

Comment:
In the future, should we be more explicit when rendering partials by adding partial: "form"?